docs/Seguimiento del equipo/Acuerdo de compromiso.mdx

```diff
@@ -43,7 +46,10 @@ Al unirme a la asignatura de Ingeniería de Software y Práctica Profesional (IS
 - Se realizarán revisiones semanales de las tareas, calificando del 0 al 10 y proporcionando una puntuación media de equipo.
 - Se establecen niveles de cumplimiento del acuerdo de compromiso.
 
-### 2.7 Cumplimiento del acuerdo
+### 2.7 *Team Practices*
+- En la asignatura se establecen unas series de prácticas establecidas en el apartado de [**Prácticas recomendadas**](./Prácticas%20recomendadas.mdx), por ello, al firmar este acuerdo me comprometo a adaptarme y seguir todo lo posible estas prácticas (Siendo motivo de aviso leve si no se siguen).
```

Comment:
Esto se ha informado al grupo y si estan de acuerdo?

Reply:
As there is no consensus, this issue will be discussed in class with the teachers.

Reply:
After the discussion in class, it was concluded that this type of tool is not always useful because they are developed for educational purposes and are designed for a very specific case, which leads to a deviation from the work methodology established by the group. On this basis, we have decided that they will only be used when the recommended practices are in line with the methodology used.